### PR TITLE
LPS-1214444 Allow set alphabetical order in SelectDDMFormField

### PR DESCRIPTION
**Pull request by robertoDiaz**
Related Issue: https://issues.liferay.com/browse/LPS-1214444

### CODE REVIEW

**Comment by liferay-forms** [comment]
ci:test:sf

**Comment by robertoDiaz** [comment]
ci:test:sf

**Comment by robertoDiaz** [comment]
ci:test:relevant

**Comment by liferay-forms** [comment]
ci:test:relevant

**Comment by liferay-forms** [comment]
ci:test:relevant

**Comment by liferay-forms** [comment]
Hey @robertoDiaz, can you please check if these failures are related to your changes? Thanks

**Comment by carolmariaabb** [comment]
Just started reviewing :)<br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by robertoDiaz** [comment]
They do not seem to be related. (errors are not in the forms portlet and seem to be related with site templates)

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-lang/src/main/resources/content/Language.properties [hunk 1]
```diff
@@ -127,6 +127,7 @@ edit-element-set=Edit Element Set
 edit-x-field=Edit {0}
 element-sets=Element Sets
 empty-list=No results were found.
+enable-alphabetical-order=Enable alphabetical order
```
Comment (carolmariaabb) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-lang/src/main/resources/content/Language.properties:130]: Hi @robertoDiaz,

Could you please change the sentence to `Order Options Alphabetically`? Thanks!

**Review by carolmariaabb — CHANGES_REQUESTED** [review]

**Comment by robertoDiaz** [comment]
ci:test:sf

**Comment by robertoDiaz** [comment]
ci:test:relevant

**Comment by liferay-forms** [comment]
Hey @robertoDiaz, can you please fix the source formatter issues and resend? Thanks

**Pull request closed without merging**